vendor/wasm/WebGL/webgl.odin

```diff
@@ -174,10 +179,6 @@ Uniform1fv :: proc "contextless" (location: i32, v: f32)       { Uniform1f(locat
 Uniform2fv :: proc "contextless" (location: i32, v: glm.vec2)  { Uniform2f(location, v.x, v.y) }
 Uniform3fv :: proc "contextless" (location: i32, v: glm.vec3)  { Uniform3f(location, v.x, v.y, v.z) }
 Uniform4fv :: proc "contextless" (location: i32, v: glm.vec4)  { Uniform4f(location, v.x, v.y, v.z, v.w) }
-Uniform1iv :: proc "contextless" (location: i32, v: i32)       { Uniform1i(location, v) }
-Uniform2iv :: proc "contextless" (location: i32, v: glm.ivec2) { Uniform2i(location, v.x, v.y) }
-Uniform3iv :: proc "contextless" (location: i32, v: glm.ivec3) { Uniform3i(location, v.x, v.y, v.z) }
-Uniform4iv :: proc "contextless" (location: i32, v: glm.ivec4) { Uniform4i(location, v.x, v.y, v.z, v.w) }
```

Comment:
Note you've just changed it to make it work for your use case and haven't touched the `UniformNfv` code above it to be "correct" either.

Reply:
Indeed, I could have also updated the `UniformNfv` bindings. Would you prefer that I include these fixes in this PR, or create a separate one ?